modules/configuration/ajax/process.php

```diff
@@ -39,6 +56,8 @@
                 array('Value' => $value),
                 array('ID' => $key)
             );
+            //delete duplicate ConfigID-value pairs 
+            $DB->run($query);
```

Comment:
I would put this query execution after the insert

Reply:
the query should be entirely removed